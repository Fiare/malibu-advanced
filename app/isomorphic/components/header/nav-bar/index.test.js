import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import { NavBar } from ".";

const renderer = new ShallowRenderer();

describe("Navbar", () => {
  it("renders the NewsHeading component with linked-story-slug", () => {
    const tree = renderer.render(<NavBar />);
    expect(tree).toMatchSnapshot();
  });
});
