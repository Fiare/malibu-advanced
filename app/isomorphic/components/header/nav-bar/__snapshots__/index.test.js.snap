// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders the NewsHeading component with linked-story-slug 1`] = `
<React.Fragment>
  <AppLogo />
  <ul
    className="styles-m__navbar__1m8_H"
  >
    <li>
      <button
        onClick={[Function]}
      >
        Login
      </button>
    </li>
  </ul>
  <NavbarSearch />
</React.Fragment>
`;
